gitops/snapshot.go

```diff
@@ -152,6 +155,16 @@ func MarkSnapshotIntegrationStatusAsInProgress(adapterClient client.Client, ctx
 	if err != nil {
 		return nil, err
 	}
+	if helpers.HasLabel(snapshot, tekton.PipelineRunEnterIntegrationSvcTimeLabel) {
+		snapshotInProgressTime := &metav1.Time{Time: time.Now()}
```

Comment:
Shouldn't be this covered by tests?

Reply:
Added. Thanks!

> Shouldn't be this covered by tests?

Added. Thanks!

Reply:
I wanted to check new coverage, but it seems codecov is experiencing some issues

```
[2023-02-07T13:39:12.767Z] ['info'] Detected GitHub Actions as the CI provider.
[2023-02-07T13:39:12.770Z] ['info'] Pinging Codecov: https://codecov.io/upload/v4?package=github-action-2.1.0-uploader-0.3.2&token=*******&branch=monitor_by_prometheus&build=4109711102&build_url=https%3A%2F%2Fgithub.com%2Fredhat-appstudio%2Fintegration-service%2Factions%2Fruns%2F4109711102&commit=24417cd2e7615ed8fb217c75ce4e9f9d47787eb6&job=Go+Test+on+Pull+Requests&pr=92&service=github-actions&slug=redhat-appstudio%2Fintegration-service&name=&tag=&flags=&parent=
[2023-02-07T13:39:13.098Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 404 - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
```

Reply:
@MartinBasti we've seen that codecov error happen before, it tends to resolve itself after retrying the action later if I remember correctly.

Reply:
First thing I did was retry, I'll retry more :D